reflection.go

```diff
@@ -21,6 +21,7 @@ var argTypeMap = map[reflect.Type]discordgo.ApplicationCommandOptionType{
 	// ???: discordgo.ApplicationCommandOptionMentionable,
 	reflect.TypeOf(0.0):                           discordgo.ApplicationCommandOptionNumber,
 	reflect.TypeOf(discordgo.MessageAttachment{}): discordgo.ApplicationCommandOptionAttachment,
+	reflect.TypeOf(uint(0)):                       discordgo.ApplicationCommandOptionInteger,
```

Comment:
For a uint, this should set the minimum value for the option to 0. I'm guessing this'll be tricky to implement, so this may just make sense to leave for me to do when I do my refactors